File: Java/BOJ/11652 카드.java
import java.io.*;
import java.util.*;
import java.math.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int N = Integer.parseInt(br.readLine());
        HashMap <Long, Integer> card = new HashMap<>();
        for (int i = 0; i < N; i++) {
        	long x = Long.parseLong(br.readLine());
        	if (card.containsKey(x)) {
        		card.put(x, card.get(x)+1);
        	}
        	else {
        		card.put(x, 1);
        	}
        }
        TreeSet<Long> cards = new TreeSet<>(card.keySet());
        
        int max = 0;
        for (Long i : card.keySet()) {
        	if (max < card.get(i)) {
        		max = card.get(i);
        	}
        }
        
        for (Long i : cards) {
        	if (card.get(i) == max) {
        		System.out.println(i);
        		break;
        	}
        }
    }
}

File: Python/BOJ/11652 카드.py
import sys
N = int(sys.stdin.readline())
card = {}
for _ in range(N):
    x = int(sys.stdin.readline())
    if x not in card :
        card[x] = 1
    else:
        card[x] += 1

n = max(card.values())
for i in sorted(card):
    if n == card[i] :
        print(i)
        break